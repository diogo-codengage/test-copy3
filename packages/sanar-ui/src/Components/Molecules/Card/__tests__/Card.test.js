import React from 'react'
import TestRenderer from 'react-test-renderer'

import ESCard from '../'

it('renders correctly', () => {
    const component = (
        <ESCard title='Card test'>
            <h1>Card</h1>
        </ESCard>
    )

    const tree = TestRenderer.create(component).toJSON()
    expect(tree).toMatchSnapshot()
})
