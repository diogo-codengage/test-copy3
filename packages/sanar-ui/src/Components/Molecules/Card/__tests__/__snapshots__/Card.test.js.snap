// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="ant-card es-card ant-card-bordered ant-card-small"
>
  <div
    className="ant-card-head"
    style={Object {}}
  >
    <div
      className="ant-card-head-wrapper"
    >
      <div
        className="ant-card-head-title"
      >
        Card test
      </div>
    </div>
  </div>
  <div
    className="ant-card-body"
    style={Object {}}
  >
    <h1>
      Card
    </h1>
  </div>
</div>
`;
